book/website/reproducible-research/licensing/licensing-ethical-source.md

```diff
@@ -0,0 +1,51 @@
+(rr-licensing-ethical)=
+# 'Ethical Source'
+
+The ethical source movement seeks to affirmatively protect specific user rights by curtailing freedom 0, the freedom to use software 'for any purpose' and prohibiting the use of the software for unethical purposes.
+Consequently, these licence by the classical definitions of free and open-source software from the FSF and OSI would not be considered free or open-source licences, they do however generally resemble them in the  other three criteria of the definition.
```

Comment:
```suggestion
Consequently, these licences by the classical definitions of free and open-source software from the FSF and OSI would not be considered free or open-source licences.
The ethical course licences do generally resemble them in the other three criteria of the definition.
```

Reply:
Sentence is quite long

Reply:
Is it 'open source' or open-source. I'm used to open source. I guess it doesn't matter too much as long as your consistent in the section/chapter!